BUILD.bazel

```diff
@@ -1326,7 +1326,7 @@ ray_cc_test(
     name = "worker_pool_test",
     size = "small",
     srcs = ["src/ray/raylet/worker_pool_test.cc"],
-    tags = ["team:core"],
+    tags = ["team:core", "no_tsan"],
```

Comment:
do we know why does tsan do not work for these tests? is it by design or there are bugs preventing them to pass tsan?

Reply:
cc @scv119 do you remember?

Reply:
I'm not exactly sure, but these tests do not pass. Skipping these tests is the same behavior as before though.